### DISCUSSION TITLE
[Feature] Copy Face Recognition to other users

### DISCUSSION THREAD

**Discussion by puertocastilla** [immich-app/immich#14411] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

For all of us who share photos with family members, a function that allows us to copy recognized faces to another user would be useful.
It would be a copy of the recognized faces with their names at the current moment, and then each user would have the possibility of continuing to modify their recognized faces, but at least they would have an initial base to use from the very beginning.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
I don't think this would be possible without copying all of the assets as well.
